.github/workflows/services-persist-tag.yml

```diff
@@ -25,6 +25,7 @@ jobs:
         run: echo "${{ inputs.service-tag }}" > ./build/${{ inputs.service-name }}/version.txt
       - name: Commit changes
         run: |
+          git pull
```

Comment:
This is enough to make sure that we get the latest changes that might have happened in the repository since the triggering of the workflow.